doc/decisions/decision_process.md

```diff
@@ -105,4 +130,8 @@ These decision PRs are also merged for documentation purposes.
 
 ## Notes
 
+- Early discussions in issues or in discussions is not prohibited.
```

Comment:
If that is the case, such issues should not be closed _arbitrarily_ by someone other than the person who opened the issue. Of course if it is clear to most or all participants that the problem is well defined and a decision PR is needed or that the proposal can not be implemented right now, then the issue can be closed.